domains/certificates/Query.API/API/Query.API/Projections/CertificatesTransferProjection.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using API.Query.API.Projections.Views;
+using CertificateEvents;
+using CertificateEvents.Exceptions;
+using Marten;
+using Marten.Events;
+using Marten.Events.Projections;
+
+namespace API.Query.API.Projections;
+
+public class CertificatesTransferProjection : IProjection
```

Comment:
There is two projections (CertificatesTransferProjection and CertificatesByOwnerProjection) handling/updating the same view (CertificatesByOwnerView). I would think that having a single projection handling a single view is easier to maintain and understand - also I am not sure, if there will be issues with both projections "fighting" about the same view.
Is it possible to have everything related to CertificatesByOwnerView in this projection (CertificatesTransferProjection)?